x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_view_in_log_pattern_analysis_action.tsx

```diff
@@ -88,21 +91,33 @@ export const useViewInLogPatternAnalysisAction = (dataViewId?: string): TableIte
         : viewInLogPatternAnalysisMessage;
 
       const clickHandler = async () => {
-        const openInLogPatternAnalysisUrl = await generateLogPatternAnalysisUrl(tableItem);
-        if (typeof openInLogPatternAnalysisUrl === 'string') {
-          await application.navigateToUrl(openInLogPatternAnalysisUrl);
+        if (!isLogPattern(tableItem)) {
+          const openInLogPatternAnalysisUrl = await generateLogPatternAnalysisUrl(tableItem);
+          if (typeof openInLogPatternAnalysisUrl === 'string') {
+            await application.navigateToUrl(openInLogPatternAnalysisUrl);
+          }
         }
       };
 
-      const isDisabled = logPatternAnalysisUrlError !== undefined;
+      const isDisabled = logPatternAnalysisUrlError !== undefined || isLogPattern(tableItem);
 
       return (
         <TableActionButton
           dataTestSubjPostfix="LogPatternAnalysis"
           iconType="logstashQueue"
           isDisabled={isDisabled}
           label={viewInLogPatternAnalysisMessage}
-          tooltipText={message}
+          tooltipText={
```

Comment:
Is the way this action is disabled different to how we disable menu items elsewhere? I see the hand cursor and the disabled font color isn't as distinctive as for other disabled actions.

Reply:
This is a strange one, I tried to tweak it already to be in line with other actions but those links are structured a bit differently and it didn't work out yet. I added an item to the issue here to not forget to revisit, but will probably do in a follow up: https://github.com/elastic/kibana/issues/167467